container/docker.make

```diff
@@ -19,7 +19,7 @@ $(DOCKER_CONFIG): | $(DOCKER)
 docker: | $(DOCKER_CONFIG)
 
 $(DOCKER_COMPOSE): | $(DOCKER) $(CURL) $(JQ)
-	sudo $(CURL) -L $(shell $(CURL) -L https://api.github.com/repos/docker/compose/releases/latest \
+	sudo $(CURL) -L $(shell $(CURL) -L https://api.github.com/repos/docker/compose/releases/tags/$(DOCKER_COMPOSE_VERSION) \
 			| jq '.assets[] | select(.name == "docker-compose-linux-x86_64").browser_download_url' \
```

Comment:
Does this query still work for selecting the right name? @JCombee recently changed the casing of `linux` to make it work for 2+.

See: https://github.com/elgentos/setup/pull/29